pxr/usd/usdUtils/CMakeLists.txt

```diff
@@ -290,6 +309,11 @@ pxr_install_test_dir(
     DEST testUsdUtilsUserProcessingFunc
 )
 
+pxr_install_test_dir(
+    SRC testenv/testUsdUtilsPackageEncapsulation
```

Comment:
nit: We generally keep the test asset directory same name as the test. So "testUsdUtilsValidators", instead of "testUtilsPackageEncapsulation".